tests/ha/single_pod_restart/test_data_pod_restart_api.py

```diff
@@ -1077,17 +1074,18 @@ def test_obj_ver_suspension_data_pod_restart(self):
         LOGGER.info("Step 4: Got versions of %s with & without specifying VersionID & verified "
                     "etags for %s.", self.object_name, self.bucket_name)
 
-        LOGGER.info("Step 5: Shutdown data pod with replica method and verify cluster & "
+        LOGGER.info("Step 5: Shutdown random data pod with replica method and verify cluster & "
```

Comment:
we are not selecting random here, isn't it?

Reply:
In setup, we have selected self.delete_pod random only
 sts = self.system_random.sample(sts_list, 1)[0]
 self.delete_pod = sts_dict[sts][-1]

Should I update ?

Reply:
Oh ok